Source/Core/UICommon/DiscordPresence.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "UICommon/DiscordPresence.h"
```

Comment:
Please add newlines between different header groups (otherwise lint will complain about the order), like this:

```c++
#include "UICommon/DiscordPresence.h"

#include <ctime>
#include <discord-rpc/include/discord_rpc.h>

#include "Core/ConfigManager.h"
```

(the convention is: header for the .cpp, system/external includes, Dolphin includes)

`<>` should also be used for the Discord header instead of `""` since it's not a Dolphin include.